src/components/MovieListSmall.js

```diff
@@ -51,11 +59,37 @@ export default function MovieListSmall(props) {
 	const objConfiguration = useStore(state => state.objConfiguration);
 	const imagesBaseUrl = objConfiguration.secure_base_url + objConfiguration.poster_sizes[1];
 
+	const arrMovieBookmarks = useStore(state => state.arrMovieBookmarks);
+	const arrTvBookmarks = useStore(state => state.arrTvBookmarks);
+
+	if (props.type === 'tv') {
+		arrData.map(item => {
+			arrTvBookmarks.map(ele => {
+				if (ele.id === item.id) {
+					item.isBookmarked = true;
+				}
+			});
+		});
+	} else {
+		arrData.map(item => {
+			arrMovieBookmarks.map(ele => {
+				if (ele.id === item.id) {
+					item.isBookmarked = true;
+				}
+			});
+		});
+	}
```

Comment:
This is called from index.js, where i use this Component twice, one time for movies, one time for tv-shows.
I check if the component is called with movie or tv data, because they are not the same.
I map over loaded data (arrData) and then check if either a movie or a tv-show i loaded is present in either the arrMovieBookmarks or the arrTvBookmarks.

If i have a hit, that item (in the corresponding arrData) is marked with isBookmarked = true, so i can show the bookmark icon in the right upper corner.

If you have a more elegant solution, i'm all ears and eyes.